Src/Base/AMReX_IntegratorBase.H

```diff
@@ -0,0 +1,185 @@
+#ifndef AMREX_INTEGRATOR_BASE_H
+#define AMREX_INTEGRATOR_BASE_H
```

Comment:
We need to include `AMReX_Config.H` here so that macros like `AMREX_PARTICLES` if defined are always available.

Reply:
Done!